app/actors/hyrax/actors/file_set_ordered_members_actor.rb

```diff
@@ -10,6 +10,7 @@ def attach_to_work(work, file_set_params = {})
         file_set.visibility = work.visibility unless assign_visibility?(file_set_params)
         work.representative = file_set if work.representative_id.blank?
         work.thumbnail = file_set if work.thumbnail_id.blank?
+        work
```

Comment:
To be consistent, this version of the `attach_to_work` method also needs to return the updated work.